backend/backend.py

```diff
@@ -17,21 +17,36 @@ class Backend(ABC):
         Handle combining the video feed with overlays and displaying, and
         recording the video feed to a file. """
 
-    def __init__(self, width: int, height: int, publish_recording_status_func: PublishFunc, exception_handler: PublishFunc):
+    def __init__(self, width: int, height: int, publish_recording_status_func: PublishFunc, publish_camera_is_online_func: PublishFunc, exception_handler: PublishFunc):
         self.width = width
         self.height = height
         self.publish_recording_status_func = publish_recording_status_func
+        self.publish_camera_is_online_func = publish_camera_is_online_func
         self.exception_handler = exception_handler
 
         self.recording = False
         self.recording_output_file = None
         self.recording_start_time = None
 
-        # time that we last called self.send_recording_status
+        # Time that we last called self.send_recording_status
         self.prev_recording_status_time = 0
         # Time between recording statuses, in seconds
         self.recording_status_interval = 60
 
+        # Time that we last called self.send_camera_is_online
+        self.prev_camera_is_online_time = 0
+        # Time between recording statuses, in seconds
+        self.camera_is_online_interval = 60
+
+    @abstractmethod
+    def _is_camera_on(self) -> bool:
+        """ Check if the camera / video feed / display is on.
+
+        Returns:
+            bool: True if is on, False otherwise
+        """
+        pass
```

Comment:
When we have a docstring comment we don't need to put `pass`. It's weird

Reply:
Done
7f1ab80

Forgot that docstrings were a multiline string rather than comments like other languages :/

Reply:
Yeah it's a little weird... No idea why its like that